GhostNetwork.Gateway/GhostNetwork.Gateway.csproj

```diff
@@ -7,6 +7,7 @@
 
     <ItemGroup>
         <PackageReference Include="Boberneprotiv.Domain" Version="0.0.3" />
+        <PackageReference Include="GhostNetwork.Profiles" Version="1.3.5" />
```

Comment:
infrastructure already has this package reference

Reply:
Yes but how I can use models from profile-service in interface that store on "Domain" project?

Reply:
profile service models should be used only in infrastructure layer